frontend/appflowy_flutter/lib/plugins/database/tab_bar/tab_bar_view.dart

```diff
@@ -132,15 +123,17 @@ class _DatabaseTabBarViewState extends State<DatabaseTabBarView> {
               builder: (context, state) =>
                   pageSettingBarExtensionFromState(state),
             ),
-            Expanded(
-              child: BlocBuilder<DatabaseTabBarBloc, DatabaseTabBarState>(
-                builder: (context, state) => PageView(
-                  pageSnapping: false,
-                  physics: const NeverScrollableScrollPhysics(),
-                  controller: _pageController,
-                  children: pageContentFromState(state),
-                ),
-              ),
+            BlocBuilder<DatabaseTabBarBloc, DatabaseTabBarState>(
+              builder: (context, state) {
+                final layout = state.tabBars[state.selectedIndex].layout;
+                return SizedBox(
+                  height: layout.pluginHeight,
+                  child: IndexedStack(
```

Comment:
Change to indexed stack since we use "jumpToPage", seems redundant to use a pageview here.